.github/workflows/publish.yml

```diff
@@ -0,0 +1,261 @@
+
+    steps:
+      - uses: azure/setup-helm@v3
+        with:
+          version: '3.12.2'
+        id: install
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 16.x
+          cache: 'yarn'
+
+      - name: Set environment variables
+        run: |
+          echo "BRANCH_NAME=$(git branch --show-current)" >> $GITHUB_ENV
+          echo "TAG=$(git tag --points-at ${{github.sha}})" >> $GITHUB_ENV
+          echo "PR_LAST_COMMIT_MESSAGE=$(git log --format=%B -n 1 ${{github.event.after}} | head -n 1)" >> $GITHUB_ENV
+
+      - uses: kanga333/variable-mapper@master
+        name: Determine target branches in wireapp/wire-builds
+        with:
+          key: '${{env.BRANCH_NAME}}'
+          map: |
+            {
+              "stefan": { "wire_builds_target_branches" : "[\"main\"]" },
+              "dev": { "wire_builds_target_branches" : "[\"main\"]" }
+            }
+
+      - name: Make target branches available as output to other jobs
+        id: output_target_branches
+        shell: bash
+        run: |
+          echo "wire_builds_target_branches: $wire_builds_target_branches"
+          echo "wire_builds_target_branches=$wire_builds_target_branches" >> $GITHUB_OUTPUT
+
+      - name: Set TITLE and BUILD_DESKTOP
+        env:
+          PR_TITLE: ${{github.event.pull_request.title || env.PR_LAST_COMMIT_MESSAGE}}
+        run: |
+          echo "TITLE=${PR_TITLE}" >> $GITHUB_ENV
+          echo "BUILD_DESKTOP=${{contains(env.TAG, 'staging') || contains(env.TAG, 'production') || contains(env.PR_LAST_COMMIT_MESSAGE, '+Desktop')}}" >> $GITHUB_ENV
+
+      - name: Print environment variables
+        run: |
+          echo -e "BRANCH_NAME = ${BRANCH_NAME}"
+          echo -e "TAG = ${TAG}"
+          echo -e "TITLE = ${TITLE}"
+          echo -e "PR_LAST_COMMIT_MESSAGE = ${PR_LAST_COMMIT_MESSAGE}"
+          echo -e "COMMIT_URL = ${COMMIT_URL}"
+          echo -e "COMMITTER = ${COMMITTER}"
+          echo -e "BUILD_DESKTOP = ${BUILD_DESKTOP}"
+
+      - name: Skip CI
+        if: |
+          contains(env.TITLE || env.PR_LAST_COMMIT_MESSAGE, 'skip ci') ||
+          contains(env.TITLE || env.PR_LAST_COMMIT_MESSAGE, '[ci skip]')
+        uses: andymckay/cancel-action@0.3
+
+      - name: Authenticate git clone
+        env:
+          GH_TOKEN: ${{secrets.OTTO_THE_BOT_GH_TOKEN}}
+        run: echo "machine github.com login ${GH_TOKEN}" > ~/.netrc
+
+      - name: Install JS dependencies
+        run: yarn --immutable
+
+      - name: Test
+        run: |
+          set -o pipefail
+          yarn test --coverage --coverage-reporters=lcov --detectOpenHandles=false 2>&1 | tee ./unit-tests.log
+
+      - name: Monitor coverage
+        uses: codecov/codecov-action@v3.1.4
+        with:
+          fail_ci_if_error: false
+          files: ./coverage/lcov.info
+          flags: unittests
```

Comment:
```suggestion
```
We don't need to monitor the coverage for this action. This will be done in the CI job (for PRs)

Reply:
Removed in fd1d2c8c6daeb79399d18353699efc1d7b86b759